verticapy/connection/connect.py

```diff
@@ -201,6 +303,18 @@ def vertica_connection(section: str, dsn: Optional[str] = None) -> Connection:
     -------
     conn
         Database connection.
+
+    Example
+    -------
+    Create a connection using the input DSN:
+
+    .. code-block:: python
+
+        from verticapy.connection import vertica_connection
+
+        vertica_connection("VerticaDSN")
+
+    ``<vertica_python.vertica.connection.Connection at 0x106526198>``
```

Comment:
.. note:: We utilize a Data Source Name (DSN) to establish the connection, which is stored in the file specified by the global variable VERTICAPY_CONNECTION. However, if you prefer a customized file located elsewhere, you can specify the file path accordingly.